docs/configuration.md

```diff
@@ -69,7 +69,29 @@ if (!hasProperty("productionBuild")) {
 }
 ```
 
-##### <your build here>
+### Variant filter
+
+You can specify a `variantFilter` on the `keeper` extension and dynamically configure which variants
```

Comment:
Accidentally included the start of this paragraph twice?

Reply:
41c9252